examples/simple-email-form/controller.php

```diff
@@ -13,7 +14,12 @@ public function __construct()
             $_SESSION[$key] = $val;
         }
 
-        if (isset($_POST['email_label']) && isset($_POST[$_POST['email_label']]) && isset($_POST['mautic_base_url']) && isset($_POST['form_id'])) {
+	if (isset($_POST['email_label']) && isset($_POST[$_POST['email_label']]) && isset($_POST['mautic_base_url']) && isset($_POST['form_id'])) {
+		
+	    //Setting Curl verbose logging to true
+	    $config = new Config;
+	    $config->setCurlVerbose(1);
```

Comment:
```suggestion
	    $config->setCurlVerbose(true);
```
Thinking about it, I know I used `1` when configuring CURL, but a bool value makes more sense.